src/react-apps/ux-editor/src/styles/toolBar.css

```diff
@@ -0,0 +1,20 @@
+.a-item {
```

Comment:
duplicate file??

Reply:
Looks like the old file structure

Reply:
Came with the merging of master into this branch. Remove? We don't have any other css-file for the toolbar.